sdk/core/ts-http-runtime/review/ts-http-runtime.api.md

```diff
@@ -46,6 +37,15 @@ export interface Agent {
     sockets: unknown;
 }
 
+// @public
+export function bearerTokenAuthenticationPolicy(options: BearerTokenAuthenticationPolicyOptions): PipelinePolicy;
+
+// @public
+export interface BearerTokenAuthenticationPolicyOptions {
+    credential: TokenCredential;
+    scopes: string | string[];
```

Comment:
should we simplify this to only `string[]`?

Reply:
Ah good catch, updated!